tortoisebeacon/tortoise_beacon.go

```diff
@@ -87,14 +87,9 @@ type TortoiseBeacon struct {
 	vrfSigner        signer
 	weakCoin         weakcoin.WeakCoin
 
-	layerMu   sync.RWMutex
-	lastLayer types.LayerID
-
-	seenEpochsMu sync.Mutex
-	seenEpochs   map[types.EpochID]struct{}
+	epoch types.EpochID
```

Comment:
I tested it on Raspberry PI 4 with ARM32 OS image, it doesn't panic. However, if we add a variable in the future that changes the alignment, this can start causing panic. As we're migrating from mutex to atomics now, I think we should move it now to the uppermost place in this `struct` and note this in a comment.